docs/HARNESSES.md

```diff
@@ -0,0 +1,142 @@
+## [🏠][home] &nbsp; → &nbsp; [Documentation][api] &nbsp; → &nbsp; **Component Test Harnesses**
+
+<details>
+  <summary>🧭 &nbsp;<b>Related topics</b></summary>
+
+> ### First Steps
+>
+> You may want to visit our [first steps page][firststeps].
+>
+> ### Writing Good Tests
+>
+> Introduction on [how to write simple, maintainable tests using ngtx][goodtests] and its features.
+>
+> ### Declarative Testing API
+>
+> Formal documentation of ngtx' [declarative testing api][declarativetests] helping you to write beautiful, simple and robust tests.
+
+---
+
+</details>
+
+&nbsp;
+
+This article explains what component test harnesses are and why they are useful for everyday testing.
+
+> ### Pro Tip!
+>
+> To make the most of it, combine them with ngtx' [declarative testing api][goodtests]!
+
+## Use Component Test-APIs (aka "Component Harnesses")
+
+Although being "DRY" is something absolutely normal in production code, this is often not true in testing code. But fact is, the same reasons behind us writing DRY code in production, should urge us to maintaining DRY in testing code as well.
```

Comment:
```suggestion
staying ["DRY"](https://en.wikipedia.org/wiki/Don't_repeat_yourself) is absolutely a standard that every software design patterns book recommends you do as a developer. This is often hard to do in writing testing code. But fact is, the same reasons behind us writing DRY code in production code, should urge us to maintaining DRY in testing code as well.
```

Reply:
I edited a bit but applied several of your improvements